biologging-sensor-data/src/components/overview/Snippet.tsx

```diff
@@ -51,8 +51,8 @@ export default function OverviewSnippet({ data }: { data: Dataset | null }) {
                 </div>
 
                 <div className="row">
-                    <div className="col-md-2 bold">Instruments:</div>
-                    <div className="col-md-9">{data?.instrumentTypes?.join(", ")}</div>
+                    <div className="col-md-2" style={bold}>Instruments:</div>
```

Comment:
I think the bold style here should be defined as it was before, not sure why it was changed. So it should be in classname as 'bold' and not style={bold}. Else the application crashes when clicking on the row

Reply:
@zrohacova Because in all other row you have style={bold}.
What you mean to click on the row, as I understand it`s not clickable...